api/src/main/java/org/pmiops/workbench/db/dao/UserServiceImpl.java

```diff
@@ -420,24 +420,6 @@ public DbUser createUser(
     return dbUser;
   }
 
-  /**
-   * Save updated dbUser object and transform ObjectOptimisticLockingFailureException into
-   * ConflictException
-   *
-   * @param dbUser
-   * @return
-   */
-  @Override
-  public DbUser updateUserWithConflictHandling(DbUser dbUser) {
```

Comment:
`updateUserWithRetries()` does this plus a lot more, so this is no longer needed.